subworkflows/functional_annotation/main.nf

```diff
@@ -43,7 +43,10 @@ workflow FUNCTIONAL_ANNOTATION {
         GFF2PROTEIN.out.proteins.splitFasta( by: params.records_per_file, file: true ),
         blastdb_ch.map{ meta, db -> db }.collect()
     )
-    INTERPROSCAN( GFF2PROTEIN.out.proteins.splitFasta( by: params.records_per_file, file: true ) )
+    INTERPROSCAN( 
+        GFF2PROTEIN.out.proteins.splitFasta( by: params.records_per_file, file: true ),
+        out_text='tsv'
```

Comment:
This is strange. This is not valid nextflow. Is there a warning in the nextflow log?

Reply:
There is no warning. I was not sure how to set this parameter. I tested it and its doing whats supposed to be. What is the official way?

Reply:
```suggestion
        'tsv'
```
Passing the string will implicitly convert it to value channel so Nextflow can use it.
I.e. this is equivalent to using
```nextflow
    INTERPROSCAN( 
        GFF2PROTEIN.out.proteins.splitFasta( by: params.records_per_file, file: true ),
        Channel.value('tsv')
    )
```